third_party/libwebrtc/webrtc/modules/bitrate_controller/nada_bandwidth_estimation.cc

```diff
@@ -153,7 +95,28 @@ void NADABandwidthEstimation::CurrentEstimate(int* bitrate,
                                               uint8_t* loss,
                                               int64_t* rtt) const {
 
-//TODO (sergio): Convince Xiaoqing to remove delay_based_bitrate_bps_ altogether: it's the same as bitrate_ !!
+// TODO (sergio): Convince Xiaoqing to remove delay_based_bitrate_bps_ altogether: 
+// it's the same as bitrate_ !!
+// 
+// [Notes: 2020-11-08]
+// Xiaoqing's observation on Mozilla behavior when USE_DELAY_BASED is active (1): 
+// 
+// In NADA-RTT mode (use_transport_cc as false), the delay_based_bitrate_bps_ is 
+// chosen instead of bitrate_, but UpdateDelayBasedEstimate() is never called so 
+// delay_based_bitrate_bps_ is stuck at default rate -- so this is undesired
+// 
+// In NADA-OWD mode (use_transport_cc as true), the delay_based_bitrate_bps_ is
+// chosen based on rate calculation from NadaOwdBwe whereas bitrate_ is 
+// still periodically updated. So reporting the bitrate as delay_based_bitrate_bps_
+// is desired.
+// 
+// Question: can we switch the flag of USE_DELAY_BASED according to the
+// configuration flag use_transport_cc?   
+//
+// Proposed alternative modification: simply override bitrate_ with input 
+// rate of the UpdateDelayBasedEstimate() function whenever it is invoked
+//  
+
```

Comment:
I checked the code.
* The use_transport_cc flag is plugged into the "rtp receiver" classes. I couldn't find the common high-level class that has access to both use_transport_cc and the classes that create the "send-side estimators". *However* even if such a class existed, I doubt it is wise to depend in "configuration" to select the mode: we don't know if the remote peer will be configured with the flag as well
* What I would propose here (similar to your "alternative modification") is that we set a boolean to "RTT-mode" initially, but as soon as we receive OWD packets, we switch it to "OWD-mode"